cdot_ppmBsm.properties

```diff
@@ -0,0 +1,35 @@
+# Configuration details for the velocity filter.
```

Comment:
cdot_** files can be removed, they are being handled by another PR for the PPM to abstract away from inclusion in the main project.

Reply:
These files have been removed.